engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/FunctionalTestMixin.scala

```diff
@@ -22,11 +22,17 @@ trait FunctionalTestMixin {
   protected def invalidTypes(typeErrors: String*): Invalid[NonEmptyList[CustomNodeError]] =
     invalid(typeErrors.toList, Nil, Nil)
 
-  protected def invalid(typeFieldErrors: List[String], missingFieldsError: List[String], redundantFieldsError: List[String]): Invalid[NonEmptyList[CustomNodeError]] = {
-    val finalMessage = OutputValidatorErrorsMessageFormatter.makeMessage(typeFieldErrors, missingFieldsError, redundantFieldsError)
+  protected def invalidRanges(rangeErrors: String*): Invalid[NonEmptyList[CustomNodeError]] =
```

Comment:
Maybe just `invalid(errors: String*)` - as I understand here we can pass any error messages?

Reply:
its just simple facade for `def invalid` which is little below and it complies convention introduced by `def invalidTypes` little above, so I believe it should stay as it is now.